Finale/Finale/Presentation/ReaditDetail/ReaditIndexSubView/LevelInfoSection.swift

```diff
@@ -54,11 +44,11 @@ struct LevelInfoSection: View {
             }
         }
         .frame(maxWidth: .infinity, alignment: .leading)
-        .padding(.bottom, 16)
+        .padding(bottom: 12, leading: 8, trailing: 8)
```

Comment:
P5: 이럴땐 아래와 같이 줘도 됩니당!
```suggestion
        .padding(horizontal: 8, bottom: 12)
```